src/main/java/util/GithubPageElements.java

```diff
@@ -16,6 +16,10 @@
     public static final String COMMITS_TAB = "commits";
     public static final String FILES_TAB = "files";
 
+    public enum PrTab {
+        DISCUSSION, COMMITS, FILES
+    }
```

Comment:
If you are using this, maybe you should to put the String above this enum as enum values? Otherwise don't use the enum at all but rather use the aboves.